cg/constants/lims.py

```diff
@@ -4,6 +4,7 @@
     "capture_kit": "Capture Library version",
     "collection_date": "Collection Date",
     "comment": "Comment",
+    "control": "control",
```

Comment:
Seems to me like we usually name the LIMS UDF with a leading capital letter:
```suggestion
    "control": "Control",
```